setup_database.py

```diff
@@ -0,0 +1,49 @@
+import os
```

Comment:
I appreciate this, but isn't it just easier to do

```
% psql < sql/*.sql
```
?

Reply:
Yes, that works too.

Here, we can include custom logic before running the SQL statement.